packages/flutter/lib/src/material/theme.dart

```diff
@@ -161,6 +161,45 @@ class Theme extends StatelessWidget {
         .resolveFrom(context);
   }
 
+  /// Retrieves the [Brightness] to use for descendant Material widgets, based
+  /// on the value of [ThemeData.brightness] in the given [context].
+  ///
+  /// If no [InheritedTheme] can be found in the given [context], or its `brightness`
+  /// is null, it will fall back to [MediaQueryData.platformBrightness].
+  ///
+  /// See also:
+  ///
+  /// * [maybeBrightnessOf], which returns null if no valid [InheritedTheme] or
+  ///   [MediaQuery] exists, instead of throwing.
```

Comment:
Since `platformBrightnessOf` does not throw, but returns a default, I think we can tweak the wording here.

```suggestion
  ///   [MediaQuery] exists.
```